J/jemalloc/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "jemalloc"
+version = v"5.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/jemalloc/jemalloc.git", "886e40bb339ec1358a5ff2a52fdb782ca66461cb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd jemalloc/
+autoconf
+
+if [[ "${target}" == "x86_64-linux-gnu" ]]; then
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls
+else
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+fi
+
+make
+make install
+
+# Rename .dll for Windows targets.
+if [[ "${target}" == *"w64"* ]]; then
+    mkdir -p ${libdir}
+    mv ${prefix}/lib/jemalloc.dll ${libdir}/libjemalloc.dll
+elif [[ "${target}" == "x86_64-linux-gnu" ]]; then
+    mv ${libdir}/libjemalloc.so.2 ${libdir}/libjemalloc.so
+fi
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(;experimental=true)
+filter!(!Sys.isfreebsd, platforms) # Should already be available on freebsd
```

Comment:
What do you mean?  I don't think we have jemalloc anywhere in the FreeBSD compiler shard:
```console
% julia-16 --compile=min -e 'using BinaryBuilderBase; BinaryBuilderBase.runshell(Platform("x86_64", "freebsd"))'
sandbox:${WORKSPACE} # qfind /opt/${target}/${target} -name "*jemalloc*"
sandbox:${WORKSPACE} #
```
so how do you think to link to it?

Reply:
Think I found a fix here.